bigtable/tests/system.py

```diff
@@ -876,10 +876,10 @@ def test_yield_rows_with_row_set(self):
 
         read_rows = self._table.yield_rows(row_set=row_set)
 
-        expected_row_keys = set([b'row_key_1', b'row_key_3', b'row_key_4',
-                                 b'row_key_5', b'row_key_6'])
-        found_row_keys = set([row.row_key for row in read_rows])
-        self.assertEqual(found_row_keys, set(expected_row_keys))
+        expected_row_keys = [b'row_key_1', b'row_key_3', b'row_key_4',
+                             b'row_key_5', b'row_key_6']
+        found_row_keys = [row.row_key for row in read_rows]
+        self.assertEqual(found_row_keys, expected_row_keys)
```

Comment:
Is the ordering guaranteed here?

Reply:
Yes.  Cloud Bigtable returns (and stores) rows in by lexicographical order.